packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,165 @@
+---
+section: PatternFly-AI
+subsection: Chatbot
+id: Overview
+sortValue: 1
+source: design-guidelines
+---
+
+import "./images.css"
+
+## Elements 
+
+<div class="ws-docs-content-img">
+![Main elements of a ChatBot.](./img/chatbot-elements.svg)
+</div>
+
+1. **Container:** The window that contains the entire ChatBot experience and all of its components.
+1. **Header:** A persistent region at the top of the ChatBot window that contains navigation, branding, and actions.
+1. **Navigation:** A menu that contains navigational options, including access to the conversation history.
+1. **Messages:** Elements of the conversation between a ChatBot and user. For more details, refer to the [ChatBot message guidelines](#messages).
+1. **Attachments:** Details about files that a user has uploaded to the ChatBot.
+1. **Footer:** A persistent region at the bottom of the ChatBot window that contains the message bar and the footnote.
+1. **Footnote (optional):** A persistent, short message that contains any legal disclaimers or important information about the ChatBot. Footnotes are optional, but strongly recommended. For more information, refer to the [ChatBot footnote guidelines](#footnotes).
+1. **Toggle:** The button that allows users to open and close the ChatBot window. When the ChatBot is opened, the toggle should appear below the ChatBot window.
+
+### Toggle 
+
+Users can toggle a ChatBot open and closed, so that conversations can persist over time and be revisited as needed. The toggle is a floating button that's placed in the bottom corner of the UI. 
+
+In addition to this floating button, you can place an icon in the masthead that acts as a toggle. 
+
+### Messages 
+
+<div class="ws-docs-content-img">
+![A basic user message, bot message, and quick reply buttons.](./img/message-elements.svg)
+</div>
+
+1. **Bot message:** Messages from the ChatBot, which are marked with an "AI" label to communicate the use of AI to users. You should choose a descriptive name for your ChatBot.
```

Comment:
1.
<img width="866" alt="Screenshot 2024-11-21 at 12 10 51 PM" src="https://github.com/user-attachments/assets/5ea03649-94f3-454f-a5ad-5b8c5a2a5b70">
 says it's the bot message, but it's actually the user message in this image.